ui/src/landing/wishTypeList/index.test.js

```diff
@@ -37,12 +37,14 @@ describe('WishList tests when all fields are valid', () => {
 
 describe('WishList tests when a field is invalid', () => {
   const selectWishType = jest.fn()
-  const validFields = jest.fn(() => ({validAge: false}))
+  const validFields = jest.fn(() => ({validAge: false, validName: false}))
   const age = '19'
+  const name = ''
 
   it('should call selectWishType with selected wish when wish is clicked', () => {
```

Comment:
I know you didn't write this title, but looking at the test it feels a little inaccurate. 

Do you mind updating this to match the intent of the test?

Reply:
Sure, I'll update it